cultivation/scripts/task_management/test_session_recovery.py

```
@@ -0,0 +1,92 @@
+                parts = line.split("\t")
+                task = parts[0].split(": ")[1]
+                start = parts[1].split(": ")[1]
+                note = parts[-1].split(": ")[1].strip()
+                starts[(task, start)] = note
+            elif "Status: END" in line:
+                parts = line.split("\t")
+                task = parts[0].split(": ")[1]
+                start = parts[1].split(": ")[1]
+                ends.add((task, start))
+    # Return any starts not closed
+    return [(task, start, note) for (task, start), note in starts.items() if (task, start) not in ends]
```

Pylint (3.3.7)

[convention] Line too long (103/100)

(C0301)